app/models/manageiq/providers/amazon/instance_types.rb

```diff
@@ -13,6 +13,30 @@
 module ManageIQ::Providers::Amazon::InstanceTypes
   # Types that are currently advertised for use
   AVAILABLE_TYPES = {
+    "t2.nano"     => {
+      :name                    => "t2.nano",
+      :family                  => "General Purpose",
+      :description             => "T2 Nano",
+      :memory                  => 0.5.gigabyte,
+      :vcpu                    => 1,
+      :ebs_only                => true,
+      :instance_store_size     => 0,
+      :instance_store_volumes  => 0,
+      :architecture            => [:i386, :x86_64],
+      :virtualization_type     => [:hvm],
+      :network_performance     => :low,
```

Comment:
@jvlcek This is weird, but I think this should be `:low_to_moderate`, as that is what's shown when launching a new instance.  I see the documentation matrix shows low, so I'm not 100% sure what's the right answer, but in the past, I've leaned toward what's shown when launching a new instance.  (Then again, I'm not sure we use this value, so it may not make a difference)